vitrina/datasets/models.py

```diff
@@ -744,6 +744,10 @@ def name(self):
         if metadata := self.metadata.first():
             return metadata.name
         return ""
+    
+    @property
+    def identifier(self) -> str | None:
+        return identifier.notation if (identifier := self.identifiers.first()) else None
```

Comment:
random picking first, maybe solution to filter by agency as well (some default agency)?